### PR TITLE
Update ttl.md to include resources: video, colab, report

### PR DESCRIPTION
**Pull request by noaleetz**
## Description

Adding link to youtube video and report + colab for programmatic TTL
## Ticket

Does this PR fix an existing issue? If yes, provide a link to the ticket here:

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/artifacts/ttl.md [hunk 1]
```diff
@@ -4,11 +4,16 @@ displayed_sidebar: default
 ---
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import { CTAButtons } from '@site/src/components/CTAButtons/CTAButtons.tsx';
 
 # Manage data retention with Artifact TTL policy
 
+<CTAButtons colabLink="https://colab.research.google.com/drive/1ZP6W4FhkmGUOPPloIHqUhGLd2vYBVvph?usp=sharing"/>
```
Comment (ngrayluna) [inline docs/guides/artifacts/ttl.md]: @noaleetz Can you double check this is the correct shareable link to use? (Want to make sure it's accessible to everyone :)

**Review by ngrayluna — CHANGES_REQUESTED** [review]
Moved around links. Added a button for the Colab link.  One comment to double-check and we should be good to go.

Note: I removed the link to the report because, it is redundant to what we have on this page...

File: docs/guides/artifacts/ttl.md (hunk 1 above)
Comment (noaleetz) [reply docs/guides/artifacts/ttl.md]: started a thread w you and Kevin Stofan here on moving this to [colab](https://weightsandbiases.slack.com/archives/C01DX8LRZEJ/p1705693980118379) in our repo - he owns the colab so he is the best person to close loop w this

Comment (noaleetz) [reply docs/guides/artifacts/ttl.md]: Hi @ngrayluna - with thomas' PR merged - it looks like this is the best link to use which points to the colab in our repo - https://colab.research.google.com/github/wandb/examples/blob/kas-artifacts-ttl-colab/colabs/wandb-artifacts/WandB_Artifacts_Time_to_live_TTL_Walkthrough.ipynb. 

Can we update and merged this PR?

**Review by noaleetz — COMMENTED** [review]
needs updated link to colab in github repo

**Pull request merged**